lib/block-supports/block-style-variations.php

```diff
@@ -394,17 +412,17 @@ function gutenberg_enqueue_block_style_variation_styles() {
 	remove_action( 'wp_enqueue_scripts', 'wp_enqueue_block_style_variation_styles' );
 }
 
-if ( function_exists( 'wp_resolve_block_style_variations_from_primary_theme_json' ) ) {
-	remove_filter( 'wp_theme_json_data_theme', 'wp_resolve_block_style_variations_from_primary_theme_json' );
+if ( function_exists( 'wp_resolve_block_style_variations_from_theme_json_default' ) ) {
+	remove_filter( 'wp_theme_json_data_theme', 'wp_resolve_block_style_variations_from_theme_json_default' );
 }
 if ( function_exists( 'wp_resolve_block_style_variations_from_theme_json_partials' ) ) {
 	remove_filter( 'wp_theme_json_data_theme', 'wp_resolve_block_style_variations_from_theme_json_partials' );
 }
 if ( function_exists( 'wp_resolve_block_style_variations_from_styles_registry' ) ) {
 	remove_filter( 'wp_theme_json_data_theme', 'wp_resolve_block_style_variations_from_styles_registry' );
 }
-if ( function_exists( 'wp_resolve_block_style_variations_from_theme_style_variation' ) ) {
-	remove_filter( 'wp_theme_json_data_user', 'wp_resolve_block_style_variations_from_theme_style_variation' );
+if ( function_exists( 'wp_resolve_block_style_variations_from_theme_json_user' ) ) {
+	remove_filter( 'wp_theme_json_data_user', 'wp_resolve_block_style_variations_from_theme_json_user' );
```

Comment:
I think we're now missing a removal of a core filter. With this rename, we remove 4 core filters but we have 5, at least we will have 5 once the `default` origin change is backported.